queries.sql

```diff
@@ -50,3 +50,13 @@ SELECT species.name, COUNT(species.name) FROM species LEFT JOIN animals ON speci
 SELECT animals.name, owners.full_name FROM owners LEFT JOIN animals ON owners.id = animals.owner_id WHERE owners.full_name = 'Jennifer Orwell' AND animals.species_id = 2;
 SELECT animals.name, owners.full_name FROM owners LEFT JOIN animals ON owners.id = animals.owner_id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
 SELECT owners.full_name, COUNT(animals.owner_id) FROM owners LEFT JOIN animals ON owners.id = animals.owner_id GROUP BY owners.full_name;
+
+SELECT id, animals_id, vets_id FROM visits WHERE vets_id = 1 ORDER BY id DESC LIMIT 1;
```

Comment:
- [ ] To get `the last animal seen by William Tatcher`, the SELECT statement should be returning the animals name and the visit's date. Check example below
```sql
SELECT animals.name, visits.visit_date
FROM visits
LEFT JOIN animals ON animals.id = visits.animal_id
LEFT JOIN vets ON vets.id = visits.vet_id
WHERE vets.name = 'William Tatcher'
ORDER BY visits.visit_date DESC
LIMIT 1;
```